coderd/database/migrations/000223_template_display_name.up.sql

```diff
@@ -0,0 +1,24 @@
+-- Update the template_with_names view by recreating it.
+DROP VIEW template_with_names;
+CREATE VIEW
+	template_with_names
+AS
+SELECT
+	templates.*,
+	coalesce(visible_users.avatar_url, '') AS created_by_avatar_url,
+	coalesce(visible_users.username, '') AS created_by_username,
+	coalesce(organizations.name, '') AS organization_name,
+	coalesce(organizations.display_name, '') AS organization_display_name,
+	coalesce(organizations.icon, '') AS organization_icon
+FROM
+	templates
+		LEFT JOIN
+	visible_users
+	ON
+		templates.created_by = visible_users.id
```

Comment:
Is it possible that the created_by user is not visible by the caller of this? I'm thinking the case where someone is removed from an org after creating a template.

Reply:
It is, but we decided awhile back that if you can read the template, you should also see who created it. So granting `read` permission on the template gives some implicit permission to reading the user's name + organization information.